### ISSUE TITLE
'cat package.json | ncu -ua' doesn't work as expected

### ISSUE THREAD

**Issue by silverwind** [tjunnone/npm-check-updates#123]
Seems my `-ua` flag is actually ignored if I pipe in a package.json.

```console
$ cat package.json | ncu -ua

The following dependencies are satisfied by their declared version range, but the installed versions are behind. You can install the latest versions without modifying your package.json by using npm update. If you want to update the dependencies in your package.json anyway, use ncu --upgradeAll.

 async           ^1.4.0  →  ^1.4.2
 clean-css       ^3.3.8  →  ^3.3.9
 detect-engine   ^1.0.2  →  ^1.1.0
 file-extension  ^1.1.3  →  ^1.1.4
 minimist        ^1.1.2  →  ^1.1.3
```

```console
$ ncu -ua

All dependencies match the latest package versions :)

Upgraded /Users/silverwind/git/droppy/package.json
```

**Comment by metaraine** [comment]
That's unfortunate. The upgrade functionality doesn't work at all if a package is piped in, because it is just string input, it doesn't know where to write it. I need to add a command-line option for specifying a package.json to upgrade. e.g. `--packageFile package.json`

**Comment by metaraine** [comment]
Let me know if you have another suggestion, otherwise I will go ahead and add that functionality.

**Comment by metaraine** [comment]
In master, waiting to publish.

**Comment by silverwind** [comment]
That'd work, I guess. If you want to go full unix style, you could do `cat package.json | ncu > package.json`, but that's a bit too much I guess. I'm fine with an option like that, thought I'm almost at the point where I switch out Grunt for a Makefile because of these issues :)

**Comment by metaraine** [comment]
You would have to add `--jsonAll` in order to `> package.json` since the default output is human-friendly messaging.

Anyway, `--packageFile` available in `v2.2.0` :).
